sql/migrations/20211115170041_world.sql

```diff
@@ -0,0 +1,223 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20211115170041');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20211115170041');
+-- Add your query below.
+
+
+-- Update Warlock Quest Chains
+-- Quest 4964 Requires 4976 AND 4962
+UPDATE `mangos`.`quest_template` SET `RequiredCondition`='537' WHERE  `entry`=4964 AND `patch`=0;

```

Comment:
Do not include the database name in the migration. People can have it named differently.